playbook/app/pb_kits/playbook/pb_bread_crumbs/_bread_crumbs.jsx

```diff
@@ -0,0 +1,55 @@
+/* @flow */
+
+import React from 'react'
+import PropTypes from 'prop-types';
+import classnames from 'classnames'
+import { globalProps } from '../utilities/globalProps.js'
+
+import {
+  buildAriaProps,
+  buildCss,
+  buildDataProps,
+} from '../utilities/props'
+
+type BreadCrumbsProps = {
+  aria?: object,
+  className?: string,
+  data?: object,
+  id?: string,
+  text?: string
+}
+const BreadCrumbs = (props: BreadCrumbsProps) => {
+  const {
+    aria = {},
```

Comment:
```suggestion
    aria = { label: 'Breadcrumb Navigation' },
```

Reply:
Almost forgot this comment.. So here if we use the `<nav>` we don't need `aria-role`, but adding `aria-label` will provide some context for the assistive device.